_opcodes/op152.html

```diff
@@ -15,8 +15,6 @@
 
 <table border="1" width="98%">
 	<colgroup>
-		<col width="5%" />
-		<col width="12%" />
 		<col width="*" />
 	</colgroup>
```

Comment:
Does a 1-column colgroup even make sense? Looks like you can remove that too.

Reply:
Done.